db/schema.rb

```diff
@@ -10,11 +10,27 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_11_19_013744) do
+ActiveRecord::Schema.define(version: 2018_11_29_231413) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "applicants", force: :cascade do |t|
+    t.string "first_name"
+    t.string "last_name"
+    t.string "passport_number"
+    t.string "email"
+  end
+
+  create_table "clients", force: :cascade do |t|
+    t.string "identity"
```

Comment:
i think it should be the other way around. An identity belongs to a client, so this coloun shouldn't be there. Instead there's a `t.references :clients` on the identities table, and identities.rb has `belong_to :client`